niet/__init__.py

```diff
@@ -179,4 +221,7 @@ def main():
     if args.in_place and infilename:
         out_file = infilename
 
+    if to_delete:
+        result = delete_objects(result, to_delete)
```

Comment:
See my previous comment

Reply:
I factorized delete_objects and delete_object into a single and more easy to use delete function